src/features/home/api/useGetSettlementList.ts

```
@@ -0,0 +1,12 @@
+import { useQuery } from '@tanstack/react-query';
+import { getSettlementList } from '@/entities/group/api/group';
+import type { SettlementSort, SettlementStatus } from '@/entities/group/model/group.type';
```

ESLint

[error] Replace `·SettlementSort,·SettlementStatus·` with `⏎··SettlementSort,⏎··SettlementStatus,⏎`

(prettier/prettier)